apps/challenges/views.py

```diff
@@ -822,6 +824,7 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
         response_data = {"error": message}
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
+    is_test_annotation_missing = False
```

Comment:
Can we also add the same check for the missing evaluation script?

Reply:
@RishabhJain2018 we won't allow challenge creation without evaluation script right? the feature we added was for updating the evaluation script correct?

Reply:
> we won't allow challenge creation without evaluation script right?

Yes!

> the feature we added was for updating the evaluation script correct?
I'm missing something here, which feature are you talking about?

Reply:
I thought we would allow creating challenge without evaluation_script similar to annotations file that's why you are asking for the check.